ratelimit/redis_test.go

```diff
@@ -87,6 +89,37 @@ func Test_clusterLimitRedis_WithPass(t *testing.T) {
 	}
 }
 
+func Benchmark_clusterLimitRedis_Allow(b *testing.B) {
+	redisAddr, done := redistest.NewBenchRedis(b)
+	defer done()
+
+	clusterClientlimit := Settings{
+		Type:       ClusterClientRatelimit,
+		Lookuper:   NewHeaderLookuper("X-Test"),
+		MaxHits:    10,
+		TimeWindow: time.Second,
+		Group:      "B",
```

Comment:
Ratelimiter implementation hashes the key and appends it to the group name:
https://github.com/zalando/skipper/blob/832dedc0262faf65c330af7b1edf7fc48f173206/ratelimit/redis.go#L139-L141
so we need to test how it performs for different group name lengths, not key lengths.

Reply:
Yes, very good catch! :-)